stable/grafana/templates/deployment.yaml

```diff
@@ -106,6 +106,10 @@ spec:
             - name: ldap
               mountPath: "/etc/grafana/ldap.toml"
               subPath: ldap.toml
+          {{- if .Values.tls.enabled }}
+            - name: tls
+              mountPath: {{ .Values.tls.certDirectory }}
```

Comment:
Maybe specify a default for this path regardless?

Reply:
Let's discuss offline